From eae6a43f764c498f1ee77feed7fa5985984ed061 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 23 Jun 2011 12:42:09 -0300
Subject: [RHEL6 qemu-kvm PATCH 094/115] ehci: trace workaround

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1308832951-8995-94-git-send-email-kraxel@redhat.com>
Patchwork-id: 28410
O-Subject: [RHEL-6.2 kvm PATCH 093/115] ehci: trace workaround
Bugzilla: 561414 632299 645351 711354
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

rhel6 trace parser is buggy can doesn't grook ")" in the string
correctly.  Sidestep this issue by using "]" instead.

RHEL6-only patch.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 trace-events |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 trace-events |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/trace-events b/trace-events
index c11e9a1..16cd286 100644
--- a/trace-events
+++ b/trace-events
@@ -86,7 +86,7 @@ disable apic_set_irq(int apic_irq_delivered) "coalescing %d"
 disable usb_ehci_reset(void) "=== RESET ==="
 disable usb_ehci_mmio_readl(uint32_t addr, const char *str, uint32_t val) "rd mmio %04x [%s] = %x"
 disable usb_ehci_mmio_writel(uint32_t addr, const char *str, uint32_t val) "wr mmio %04x [%s] = %x"
-disable usb_ehci_mmio_change(uint32_t addr, const char *str, uint32_t new, uint32_t old) "ch mmio %04x [%s] = %x (old: %x)"
+disable usb_ehci_mmio_change(uint32_t addr, const char *str, uint32_t new, uint32_t old) "ch mmio %04x [%s] = %x [old: %x]"
 disable usb_ehci_usbsts(const char *sts, int state) "usbsts %s %d"
 disable usb_ehci_state(const char *schedule, const char *state) "%s schedule %s"
 disable usb_ehci_qh(uint32_t addr, uint32_t next, uint32_t c_qtd, uint32_t n_qtd, uint32_t a_qtd, int rl, int mplen, int eps, int ep, int devaddr, int c, int h, int dtc, int i) "QH @ %08x: next %08x qtds %08x,%08x,%08x - rl %d, mplen %d, eps %d, ep %d, dev %d, c %d, h %d, dtc %d, i %d"
-- 
1.7.3.2

